### ISSUE TITLE
burthorpe object blocking

### ISSUE THREAD

**Issue by GeniuSKilleR95** [itsdax/Runescape-Web-Walker-Engine#164]
[14:44:55] [DaxWalker] Generating path: {"player":{"attack":45,"defence":32,"strength":33,"hitpoints":29,"ranged":20,"prayer":50,"magic":30,"cooking":16,"woodcutting":15,"fletching":1,"fishing":18,"firemaking":1,"crafting":36,"smithing":29,"mining":34,"herblore":17,"agility":40,"thieving":30,"slayer":19,"farming":1,"runecrafting":9,"hunter":14,"construction":1,"setting":[{"key":10,"value":0},{"key":11,"value":5},{"key":17,"value":14},{"key":32,"value":0},{"key":63,"value":6},{"key":68,"value":16},{"key":71,"value":0},{"key":101,"value":74},{"key":111,"value":9},{"key":116,"value":0},{"key":131,"value":9},{"key":144,"value":0},{"key":145,"value":0},{"key":150,"value":160},{"key":165,"value":30},{"key":176,"value":0},{"key":179,"value":0},{"key":212,"value":0},{"key":273,"value":0},{"key":299,"value":0},{"key":302,"value":61},{"key":307,"value":110},{"key":314,"value":0},{"key":335,"value":0},{"key":347,"value":0},{"key":351,"value":33552394},{"key":365,"value":0},{"key":371,"value":0},{"key":387,"value":0},{"key":399,"value":0},{"key":425,"value":0},{"key":437,"value":413290},{"key":440,"value":0},{"key":482,"value":0},{"key":622,"value":0},{"key":655,"value":0},{"key":671,"value":0},{"key":705,"value":0},{"key":794,"value":0},{"key":810,"value":33554432},{"key":869,"value":0},{"key":896,"value":0},{"key":964,"value":0},{"key":970,"value":0},{"key":1630,"value":7218},{"key":1671,"value":0},{"key":1672,"value":16384}],"varbit":[{"key":192,"value":0},{"key":199,"value":0},{"key":357,"value":18},{"key":2310,"value":0},{"key":2328,"value":0},{"key":3534,"value":0},{"key":3618,"value":0},{"key":3741,"value":0},{"key":4494,"value":0},{"key":4538,"value":0},{"key":4566,"value":0},{"key":4885,"value":0},{"key":4895,"value":200},{"key":4897,"value":1},{"key":5027,"value":0},{"key":5087,"value":0},{"key":5088,"value":0},{"key":5089,"value":0},{"key":5090,"value":0},{"key":5800,"value":0},{"key":5810,"value":0},{"key":6104,"value":0},{"key":7255,"value":0},{"key":9016,"value":0},{"key":10150,"value":0},{"key":13903,"value":0}],"member":true,"equipment":[{"key":1727,"value":1},{"key":1381,"value":1},{"key":7392,"value":1},{"key":1033,"value":1}],"inventory":[]},"requests":[{"start":{"x":3225,"y":3219,"z":0},"end":{"x":3045,"y":4974,"z":1}},{"start":{"x":2915,"y":3507,"z":0},"end":{"x":3045,"y":4974,"z":1}}]}
[14:44:56] [Walker Engine] PATH_DEBUG[ 2913, 3510, 0) -> OBJECT_BLOCKING -> (2913, 3511, 0) ]
[14:44:56] [Walker Engine] Handling Object...
[14:44:56] [Object Handler] Detected: 
[14:44:56] [Object Handler] Detected: 
[14:44:56] [Walker Engine] Failed attempt on action.
[14:44:57] [Walker Engine] PATH_DEBUG[ 2913, 3510, 0) -> OBJECT_BLOCKING -> (2913, 3511, 0) ]
[14:44:57] [Walker Engine] Handling Object...
[14:44:57] [Object Handler] Detected: 
[14:44:57] [Object Handler] Detected: 
[14:44:59] [Walker Engine] Failed attempt on action.
[14:45:01] [Walker Engine] PATH_DEBUG[ 2913, 3510, 0) -> OBJECT_BLOCKING -> (2913, 3511, 0) ]
[14:45:01] [Walker Engine] Handling Object...
[14:45:01] [Object Handler] Detected: 
[14:45:01] [Object Handler] Detected: 
[14:45:02] [Walker Engine] Failed attempt on action.
[14:45:05] [Walker Engine] Too many failed attempts

**Comment by GeniuSKilleR95** [comment]
another one at catherby 

[20:59:14] [Walker Engine] Randomize(2760,3424,0) -> (2761,3425,0)
[20:59:17] [Walker Engine] Randomize(2766,3424,0) -> (2766,3423,0)
[20:59:21] [Walker Engine] Randomize(2772,3425,0) -> (2772,3426,0)
[20:59:24] [Walker Engine] PATH_DEBUG[ 2778, 3427, 0) -> OBJECT_BLOCKING -> (null [2779, 3427, 0] ) ]
[20:59:24] [Walker Engine] Handling Object...
[20:59:24] [Object Handler] Detected: 
[20:59:25] [Walker Engine] Failed attempt on action.
[20:59:26] [Walker Engine] PATH_DEBUG[ 2778, 3427, 0) -> OBJECT_BLOCKING -> (null [2779, 3427, 0] ) ]
[20:59:26] [Walker Engine] Handling Object...
[20:59:26] [Object Handler] Detected: 
[20:59:27] [Walker Engine] Failed attempt on action.
[20:59:29] [Walker Engine] Too many failed attempts

**Comment by FALSkills** [comment]
I've fixed the Burthrope issue and also added the Hero's Guild, but I don't see an issue with collision at the Catherby location.  I re-did the collision at that area nonetheless, but there is no object blocking at those tiles and so I'm unsure why the walker engine believed there was an object there.  I just tested a path there on my end and it worked without issues.

Updates will be live with the next deployment cycle
